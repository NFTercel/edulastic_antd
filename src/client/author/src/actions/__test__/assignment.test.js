import {
  addAssignmentAction,
  setAssignmentAction,
  updateAssignmentAction,
  fetchAssignmentsAction,
  deleteAssignmentAction
} from "../assignment";

describe("Assignment", () => {
  it("add assignment should return an action", () => {
    expect(addAssignmentAction({})).toMatchSnapshot();
  });

  it("set assignment should return an action", () => {
    expect(setAssignmentAction({})).toMatchSnapshot();
  });

  it("update assignment should return an action", () => {
    expect(updateAssignmentAction({})).toMatchSnapshot();
  });

  it("fetch assignment should return an action", () => {
    expect(fetchAssignmentsAction()).toMatchSnapshot();
  });

  it("delete assignment should return an action", () => {
    expect(deleteAssignmentAction(1)).toMatchSnapshot();
  });
});
