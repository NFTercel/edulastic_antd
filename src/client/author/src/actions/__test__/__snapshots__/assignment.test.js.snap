// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Assignment add assignment should return an action 1`] = `
Object {
  "payload": Object {
    "data": Object {},
  },
  "type": "[assignments] add assignment",
}
`;

exports[`Assignment delete assignment should return an action 1`] = `
Object {
  "payload": Object {
    "id": 1,
  },
  "type": "[assignments] delete assignment",
}
`;

exports[`Assignment fetch assignment should return an action 1`] = `
Object {
  "type": "[assignments] fetch assignments",
}
`;

exports[`Assignment set assignment should return an action 1`] = `
Object {
  "payload": Object {
    "data": Object {},
  },
  "type": "[assignments] set assignment",
}
`;

exports[`Assignment update assignment should return an action 1`] = `
Object {
  "payload": Object {
    "data": Object {},
  },
  "type": "[assignments] update assignment",
}
`;
